### DISCUSSION TITLE
No user IDs output from tutorial

### DISCUSSION THREAD

**Discussion by kevfrost** [opensafely/documentation#203] [Q&A] [answered]
Hi, I have Docker running, the Git libraries pulled down, everything runs smoothly in Anaconda Prompt (under administratior) but...

The output file is 

"patient_id"

Please help.

**Comment by wjchulme** [comment, marked as answer]
Thanks for pointing this out. This is a problem with the documentation rather than with the cohort extractor. [This commit](https://github.com/opensafely/documentation/pull/204/commits/68b36dce762b682030d4871c755a7eca82354d79) hopefully clears things up. But to be clear: when the study definition doesn't define any variables the `input.csv` file is as you've described -- no data, just a `patient_id` heading. It doesn't create any actual patient IDs.

**Reply by kevfrost** [reply]
Many thanks for coming back to me.  With a table that only has the header row extracted, this was a major worry that I was unable to use it.
